sass/_Variables.scss

```diff
@@ -4,6 +4,19 @@
 $coveo-orange: #f58020;
```

Comment:
Can colours like `color-orange` and `color-active-orange` be removed after the changes in the above file? Or are they still being used in other places?